updater/src/lib.rs

```diff
@@ -16,23 +16,26 @@
 
 //! Updater for Parity executables
 
-#[macro_use] extern crate log;
+#[macro_use]
```

Comment:
We usually have a convention of having separate section for `macro_use` extern crates.